scripts/remote_setup

```diff
@@ -35,7 +35,7 @@ fi
 
 # Copy local repo to machine
 echo "Transferring files..."
-rsync -a --progress --exclude=.venv,.git $PWD $CLOUD_USER@$CLOUD_ADDRESS:
+rsync -a --progress --exclude='.venv' --exclude='.git' $PWD $CLOUD_USER@$CLOUD_ADDRESS:
```

Comment:
We talked about also transferring `.git` over, right?

Reply:
Hmm ... git actually takes a while to transfer. I think what we actually want is some way to be able to push changes from the VM to git. Does it make sense to keep transferring the whole git repo before we have a way to push it? Can we save ourselves the transfer in the meantime, and add it back when we implement the mechanism to push commits from the VM?

Reply:
Good point.